run_simulation.py

```diff
@@ -52,16 +54,18 @@
                     help='length of the latent space features')
 parser.add_argument('-ve', '--vae_epochs', default=60,
                     help='how many epochs for VAE models')
-parser.add_argument('-ae', '--adage_epochs', default=100,
+parser.add_argument('-ae', '--adage_epochs', default=60,
```

Comment:
Just curious why did the number of epochs drop to 60?

Reply:
There is not much difference between 60 and 100 (for the simulation data at least) so I am using 60 to make things quicker